citar-filenotify.el

```diff
@@ -129,11 +134,7 @@ watches aren't added.  This means a mode hook containing this
 function can run several times without adding duplicate watches."
   (when (eq 'uninitialized citar-filenotify--local-watches)
     (citar-filenotify--add-local-watches))
-  (add-hook 'kill-buffer-hook
-            (lambda ()
-              (mapc #'file-notify-rm-watch citar-filenotify--local-watches)
-              (setq citar-filenotify--local-watches 'uninitialized))
-            nil t))
+  (add-hook 'kill-buffer-hook #' citar-filenotify-rm-local-watches nil t))
```

Comment:
```suggestion
  (add-hook 'kill-buffer-hook #'citar-filenotify-rm-local-watches nil t))
```
There is space between the quote and function name.

Reply:
Oops; fixed!